projects/step-frontend/src/lib/components/main-view/main-view.component.scss

```diff
@@ -1,3 +1,13 @@
 #mainView {
   height: 100%;
 }
+
+.user {
```

Comment:
ambigous class name. I'd rather use "user-label", "user-section" or something similar depending on what you display in current-user-avatar